### ISSUE TITLE
Use git tags

### ISSUE THREAD

**Issue by rugk** [TeamWin/Team-Win-Recovery-Project#621]
Could you please use git tags [to mark](https://help.github.com/articles/creating-releases/) your [releases](https://github.com/TeamWin/Team-Win-Recovery-Project/releases).

**Comment by CaptainThrowback** [comment]
Every release there's a commit with the release version.

**Comment by rugk** [comment]
This still is no argument against git tags. The listing on GitHub is e.g. an argument in favor of them.

**Comment by hlovdal** [comment]
Yes, please do use git tags. There are tons of benefits and I really cannot see any reason for not using them.
